src/PatientChecking/PatientChecking/Services/ServiceModels/Address.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+
+#nullable disable
+
+namespace PatientChecking.Services.ServiceModels
+{
+    public partial class Address
+    {
+        public int AddressId { get; set; }
+        public int TypeAddress { get; set; }
+        [Column("Address")]
+        public string Address1 { get; set; }
```

Comment:
Consider to change this to `StreetLine` and update the column in Database. We can mark it as TODO and will resolve in the next tickets